docs/source/algorithms/RefineSatellitePeaks-v1.rst

```diff
@@ -0,0 +1,80 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+RefineSatellitePeaks can be used to refine the locations of "satellite" peaks
+occurring at fractional HKL locations in reciprocal space. RefineSatellitePeaks
+takes a MDWorkspace of experimental data, a PeaksWorkspace containing the
+locations of peaks with integer HKL, and another PeaksWorkspace containing a
+subset of peaks found at fractional HKL. 
+
+For each satellite peak the difference between the nearest integer peak and
```

Comment:
should state here by what measurement you are using for the "difference". Same with "Offsets" below.